cost/aws/idle_compute_instances/idle_compute_instances.pt

```diff
@@ -795,12 +795,18 @@ EOS
       field "cpu_average" do
         label "CPU Average %"
       end
+      field "instanceType" do
+        label "Instance Type"
+      end
```

Comment:
Curt would like the recommendations to have a generic "Resource Type" field, so how about we scrape that from a standard `resourceType` field in these usage recommendations?

Currently, they may have an `instanceType` or `volumeType` field.
We can either rename those to `resourceType`, while still keeping the field "label" as "Instance Type" or "Volume Type".
Or we can add `resourceType` as an extra field, in addition to `instanceType` or `volumeType`.